VogtPayroll2/Employee.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    abstract class Employee
+    {
+        //For simplicity, use a constant tax rate of 30% to compute the tax amount
+        protected string _name;
+        protected int _hoursWorked;
+        protected decimal _payRate;
```

Comment:
Once you remove the `payRate` parameter from `GetOvertimePay()`, you should be able to remove the `_payRate` here and move it to `SalaryEmployee` instead.